src/containers/ProgrammePage/ProgrammePage.jsx

```diff
@@ -88,6 +94,14 @@ const ProgrammePage = ({ match, locale, t }) => {
 
 ProgrammePage.getDocumentTitle = getDocumentTitle;
 
+ProgrammePage.getDimensions = props => {
+  return getAllDimensions(
+    { subject: { name: getProgrammeName(props) } },
+    undefined,
+    false,
+  );
```

Comment:
Du må eksplisitt sende propsene match og locale til getProgrammeName, ellers får du feil her og '' returneres som navn.
```suggestion
ProgrammePage.getDimensions = props => {
  const { match, locale } = props;
  return getAllDimensions(
    { subject: { name: getProgrammeName(match, locale) } },
    undefined,
    false,
  );
```

Reply:
Ja, eg hadde et objekt der først. Glømte å endre det her.